### DISCUSSION TITLE
Calculate Affine transformation matrix with redundant points

### DISCUSSION THREAD

**Discussion by JY2024Aug** [chrvadala/transformation-matrix#106] [Q&A]
First, thank you very much for sharing your code here! I have one question. The code calculates the affine transformation matrix with 3 pairs of points and works great. Is there a version of this code that can take additional points, and apply Least Squares to generate the transformation parameters and report residuals as well as Root Mean Square Error (RMSE)? Many thanks! Jun

**Comment by chrvadala** [comment]
Hello, currently this feature is not available.
Every available function is described on the documentation, but what you're asking is not there.
Feel free to submit a pull request.
